automatic/ffmpeg-full/ffmpeg-full.nuspec

```diff
@@ -0,0 +1,43 @@
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>ffmpeg-full</id>
+    <version>4.4</version>
+    <title>FFmpeg</title>
+    <owners>chocolatey-community</owners>
+    <authors>FFmpeg Contributors</authors>
+    <licenseUrl>http://www.ffmpeg.org/legal.html</licenseUrl>
+    <projectUrl>http://www.ffmpeg.org/</projectUrl>
+    <iconUrl>https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-coreteampackages@edba4a5849ff756e767cba86641bea97ff5721fe/icons/ffmpeg.svg</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description><![CDATA[
+FFmpeg is the leading multimedia framework, able to decode, encode, transcode, mux, demux, stream, filter and play pretty much anything that humans and machines have created. It supports the most obscure ancient formats up to the cutting edge. No matter if they were designed by some standards committee, the community or a corporation. It is also highly portable: FFmpeg compiles, runs, and passes our testing infrastructure FATE across Linux, Mac OS X, Microsoft Windows, the BSDs, Solaris, etc. under a wide variety of build environments, machine architectures, and configurations.
+
+The FFmpeg project tries to provide the best technically possible solution for developers of applications and end users alike. To achieve this we combine the best free software options available. We slightly favor our own code to keep the dependencies on other libs low and to maximize code sharing between parts of FFmpeg. Wherever the question of "best" cannot be answered we support both options so the end user can choose.
+
+Security is a high priority and code review is always done with security in mind. Though due to the very large amounts of code touching untrusted data security issues are unavoidable and thus we provide as quick as possible updates to our last stable releases when new security issues are found.
+
+## Notes
+
+* This package is for the stable static full ffmpeg version.
+* With version 4.4 the format of the version-number changed up until the next two releases it is expected that the flag `--version` is broken
+
+]]></description>
+    <summary>FFmpeg is a complete, cross-platform solution to record, convert and stream audio and video. It includes libavcodec – the leading audio/video codec library.</summary>
+    <releaseNotes>https://ffmpeg.org/index.html#news</releaseNotes>
+    <copyright />
+    <language>en-US</language>
+    <tags>ffmpeg libavcodec foss cli audio video convert cross-platform audio video media full</tags>
+    <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-coreteampackages/tree/master/automatic/ffmpeg-full</packageSourceUrl>
+    <projectSourceUrl>https://git.ffmpeg.org/ffmpeg.git</projectSourceUrl>
+    <docsUrl>https://ffmpeg.org/ffmpeg.html</docsUrl>
+    <dependencies>
+      <dependency id="chocolatey" version="0.10.5" />
```

Comment:
is this really necessary?
(if it has been copied from the old package, please ditch this dependency)